src/search/documents/person.py

```diff
@@ -64,7 +64,22 @@ def get_queryset(
         )
 
     def prepare_headline(self, instance) -> dict[str, str]:
-        return {"title": instance.headline}
+        headline = instance.headline
+
+        # Normalize headline to object format (mapping expects object with title field)
+        if isinstance(headline, dict):
+            # Already in object format: {"title": "...", "isPublic": true}
+            # Ensure title field exists, add empty string if missing
+            if "title" not in headline:
+                headline = headline.copy()
+                headline["title"] = ""
+            return headline
+        elif isinstance(headline, str):
+            # Convert string to object format
+            return {"title": headline}
+        else:
+            # Null or other type - use empty object
+            return {"title": ""}
```

Comment:
It appears that an older implementation created headline values in the database in object structure, but newer records use plain strings. We should just run the management command to update the object structured headlines, which would make this code unnecessary.

Reply:
Alternatively, since this is already broken, it's okay to keep both in one PR, but remove the unnecessary code that manages the object-structured headline.